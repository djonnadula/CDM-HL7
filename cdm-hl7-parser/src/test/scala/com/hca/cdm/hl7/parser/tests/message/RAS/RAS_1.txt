MSH|^~\&|COCQA1A|COCQA1A|OV|OV|201611090911||RAS^O17|IP_COCQ1A_RAS_PHAQ1A.161017.4458|T|2.5||KYA^04446|ER|NE
PID|1||J000424764^^^COCQA1A^MRN|J501375|NICK^AMY||19790222|F||||||||||J00021035867
PV1|1|I|J.PSY^J.104^A^COCQA1A|EM|||DOCMMW^WILLIAMS^DOCTOR|||||||||||IN|||||||||||||||||||||||ADM
ORC|RE|128412|040241||AC||500^Q2H^Q2H||201611090911|||DOCMMW|||201611090845
RXA|1|1|201611090911|201611090845|APTI100S^TIPRANAVIR/VITAMIN E TPGS 100 MG/ML ML^L^00597000201^APTIVUS 100 MG/ML SOLUTION^NDC|5|MG|500^1||1PDDJX7355|||||||||N^N|CP|D|||||MED
RXR|PO^ORAL