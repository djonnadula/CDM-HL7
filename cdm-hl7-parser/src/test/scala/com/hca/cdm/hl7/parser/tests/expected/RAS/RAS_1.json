{
  "0000.COMN": {
    "sending_facility": "COCQA1A",
    "message_control_id": "IP_COCQ1A_RAS_PHAQ1A.125124.24362345",
    "medical_record_num": "A1352352",
    "medical_record_urn": "AG125124",
    "patient_account_num": "J15132542143"
  },
  "0001.MSH": {
    "field_separator": "|",
    "encoding_characters": "^~\\&",
    "sending_application": "COCQA1A",
    "sending_facility": "COCQA1A",
    "receiving_application": "OV",
    "receiving_facility": "OV",
    "date_time_of_message": "201497870911",
    "message_type": {
      "message_code": "RAS",
      "trigger_event": "O17"
    },
    "message_control_id": "IP_COCQ1A_RAS_PHAQ1A.125124.24362345",
    "processing_id": "T",
    "version_id": "2.5",
    "continuation_pointer": {
      "network_id": "KYA",
      "unit_number": "04446"
    },
    "accept_acknowledgement_type": "ER",
    "application_acknowledgement_type": "NE"
  },
  "0002.PID": {
    "set_id": "1",
    "patient_id_list": {
      "id_num": "A1352352",
      "assigning_authority": "COCQA1A",
      "id_type_code": "MRN"
    },
    "alternate_patient_id": "AG125124",
    "patient_name": {
      "family_name": "LOWRIDER",
      "given_name": "GARY"
    },
    "date_time_of_birth": "19871651",
    "administrative_sex": "F",
    "patient_account_num": "J15132542143"
  },
  "0003.PV1": {
    "set_id": "1",
    "patient_class": "I",
    "assigned_patient_location": {
      "point_of_care": "J.PSY",
      "room": "J.104",
      "bed": "A",
      "facility": "COCQA1A"
    },
    "admission_type": "EM",
    "attending_doctor": {
      "person_id": "ATQEFADS",
      "family_name": "FREDERIC",
      "given_name": "YARBOWLER"
    },
    "patient_type": "IN",
    "account_status": "ADM"
  },
  "0004.ORC": {
    "order_control": "RE",
    "placer_order_num": "128412",
    "filler_order_num": "040241",
    "order_status": "AC",
    "quantity_timing": {
      "quantity": "500",
      "interval": "Q2H",
      "duration": "Q2H"
    },
    "date_time_of_transaction": "201497870911",
    "ordering_provider": "ATQEFADS",
    "order_effective_date_time": "201497870845"
  },
  "0005.RXA": {
    "give_sub_id_counter": "1",
    "administration_sub_id_counter": "1",
    "date_time_start_of_administration": "201497870911",
    "date_time_end_of_administration": "201497870845",
    "administered_code": {
      "id": "APTI100S",
      "text": "TIPRANAVIR/VITAMIN E TPGS 100 MG/ML ML",
      "name_of_coding_system": "L",
      "alternate_id": "00597000201",
      "alternate_text": "APTIVUS 100 MG/ML SOLUTION",
      "name_of_alternate_coding_system": "NDC"
    },
    "administered_amt": "5",
    "administered_units": "MG",
    "administered_dosage_form": {
      "id": "500",
      "text": "1"
    },
    "administering_provider": "1PDDJX7355",
    "indication": {
      "id": "N",
      "text": "N"
    },
    "completion_status": "CP",
    "action_code": "D",
    "pharmacy_order_type": "MED"
  },
  "0006.RXR": {
    "route": {
      "id": "PO",
      "text": "ORAL"
    }
  }
}